views/partials/jira-configuration-error-summary.hbs

```diff
@@ -0,0 +1,43 @@
+<!-- Error Summary modal-->
+<div class='jiraConfiguration__errorSummaryModal' id="error-summary-modal-{{id}}">
+  <div class='jiraConfiguration__errorSummaryModalContent'>
+    <div class="jiraConfiguration__errorSummaryModal__header__container">
+      <h3>Errors occurred during the backfill</h3>
+      <span id="close-error-summary" data-installation-id="{{id}}" class="jiraConfiguration__syncErrorSummaryModal__closeBtn aui-icon aui-iconfont-close-dialog"></span>
+    </div>
+    <div class="jiraConfiguration__errorSummaryModal__header__content">
+      <p>The backfill finished but following entities were not backfilled. This may be because of permission issues, GitHub API rate-limiting and network outages. Learn more
+        You could continue to use the integration and still view the backfilled data in Jira, while the app attempts to reprocess these entities.</p>
+    </div>
+
+    <div class="error-summary-border-bottom">
+      <div class="error-summary-errors-header error-summary-border-bottom">
+        Description
+      </div>
+      {{#if syncErrors.PERMISSIONS_ERROR}}
+      <div>
+        <b>{{syncErrors.PERMISSIONS_ERROR}}</b> tasks have not been backfilled. Please
```

Comment:
Should it be repo or task?

Reply:
Yeah its tricky, probably repo because task is meaningless to the customer right? I took this from the existing warning.... but we will fix copy as ARC-2039 so will keep this flag off until thats sorted